hack/verify-configmaps.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+bash hack/generate-configmap.sh && git diff --exit-code
```

Comment:
nit: no need for the explicit `bash`; you have the `#!` shebang in the script already and the executable bit is set for the Git blob.

Reply:
Thank you! 

Fixed in https://github.com/openshift/cluster-update-keys/commit/35be0a0f942954370de1b6350612778ae6174203#diff-0991b68aa553592a457060117d2571d543f4c1cee103c6b03c3b72cd686ffa80R3.